docs/source/blocks/core/grid.md

```diff
@@ -0,0 +1,148 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Grid Block description and developer notes"
+    "property=og:description": "Volto Grid Block description and developer notes"
+    "property=og:title": "Volto Grid Block"
+    "keywords": "Volto, React, blocks, grid, container, Plone"
+---
+
+(grid-block)=
+
+# Grid Block
+
+```{versionadded} Volto 17.0.0-alpha.10
+```
+
+Volto comes with a Grid block, building upon the primitive of a container block, it is a block that contains other blocks.
```

Comment:
I would not mention the container block in the first sentence; that's an implementation detail and not relevant to users of the block. You could mention it further down.

Reply:
ok, but remember that this is dev docs, not user docs.

Reply:
I know, I was just thinking there's a difference between a developer who wants to configure the existing block and a developer who wants to build one like it.

I would at least rephrase it slightly to:
Volto comes with a Grid block, which is a block that contains other blocks arranged in a horizontal row. It is implemented using the Container block primitive.